docs/getting-started/provision-infrastructure.md

```diff
@@ -7,252 +7,274 @@ indent: true
 
 # Provision Infrastructure
 
-Crossplane allows you to provision infrastructure anywhere using the Kubernetes
-API. Once you have [installed a provider] and [configured your credentials], you
-can create any infrastructure currently supported by the provider. Let's start
-by provisioning a database on your provider of choice.
-
-Each provider below offers their own flavor of a managed database. When you
-install a provider it extends Crossplane by adding support for several "managed
-resources". A managed resource is a cluster-scoped Kubernetes custom resource
-that represents an infrastructure object, such as a database instance.
+Composite resources (XRs) are always cluster scoped - they exist outside of any
+namespace. This allows an XR to represent infrastructure that might be consumed
+from several different namespaces. This is often true for VPC networks - an
+infrastructure operator may wish to define a VPC network XR and an SQL instance
+XR, only the latter of which may be managed by application operators. The
+application operators are restricted to their team's namespace, but their SQL
+instances should all be attached to the VPC network that the infrastructure
+operator manages. Crossplane enables scenarios like this  by allowing the
+infrastructure operator to offer their application operators a _composite
+resource claim_ (XRC). An XRC is a namespaced proxy for an XR; the schema of an
```

Comment:
I think it's worth mentioning similarity with PVC and PV. I use that a lot when I'm explaining the relationship between XRC and XR and it really helps.

Reply:
I think it might be useful for folks to draw comparisons, but I also am cognizant of the fact that the mechanics of how XR / XRC works diverges pretty significantly from PV / PVC.

Reply:
I could go either way on this; @hasheddan feel free to make the call.